code/processes/tickerlogreplay.q

```diff
@@ -187,6 +194,13 @@ finishreplay:{[h;p;td]
  .save.postreplay[h;p];
  }
 
+// takes in log file directories made with segmented tickerplant
+expandstplogs:{[logdirectories] // always a list
+  $[`~tplogdir;
+    raze ` sv' logdirectories,/:key logdirectories:hsym first logdirectories;
+    `$raze pars,/:'string key each `$pars:string[logdirectories],'"/"]
```

Comment:
From what I understand you either return from the first logdirectory or all of them, this change select between first and all directories before creating all filepaths.
```suggestion
  {` sv'raze x,/:'key each x}$[`~tplogdir;{enlist first x};]hsym logdirectories
```